clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -635,4 +642,26 @@ message HitsResult {
     TotalHits totalHits = 3;
     //Ordered hits with scoring/sorting info and retrieved fields.
     repeated SearchResponse.Hit hits = 4;
+}
+
+// Specify how to highlight matched text in SearchRequest
+message Highlight {
+
+    enum Type {
+        DEFAULT = 0;
+        FAST_VECTOR = 1;
+    }
+
+    message Settings {
+        Type highlighter_type = 1;    // Specify type of highlighter to use. Ignored right now in nrtsearch.
```

Comment:
Directly changing all names in an incompatible way would make it hard to upgrade because the clientlib won't be able to support both old and new formats. If we want to change this later we would have to create new objects and methods, replace the old methods and objects with the new ones and then remove the old methods. This phase would involve inconsistency anyway. I see two options for this:
1. We just stick to camel case forever
2. We do an upgrade as specified above. In that case we would still have the inconsistency
If we are hoping to change to the correct format eventually, IMO it would be better to start naming things correctly right now.